app/helpers/quotes_helper.rb

```diff
@@ -0,0 +1,2 @@
+module QuotesHelper
```

Comment:
What was your thought for this one?

Reply:
It's a leftover from using `$rails g resource Quotes` when I should have (in hindsight) just made the files individually.